x-pack/plugins/security/public/management/roles/edit_role/edit_role_page.test.tsx

```diff
@@ -82,12 +78,6 @@ const buildFeatures = () => {
   ] as Feature[];
 };
 
-const buildRawKibanaPrivileges = () => {
```

Comment:
note: consolidating this logic to use the existing fixture that other tests are using